app/views/fines/_payment.html.erb

```diff
@@ -1,20 +1,20 @@
 <li class="list-group-item">
   <div class="row d-flex flex-wrap position-relative">
     <div class="d-flex flex-row flex-grow-1 justify-content-start w-100 row col-md-12 col-lg-6">
-      <div class="col-4 col-md-4 order-md-first description">
+      <div class="col-4 col-md-4 order-md-first bill_description" data-sort-bill_description="<%= payment.bill_description %>" >
```

Comment:
Can we do something like we're doing on the other models (using a `sort_key` method that knows how to sort + do fallbacks to other columns)?

Reply:
@jvine what do you think of these secondary sort options? The first one is the chosen sort option and the ones after the secondary:

```
[**payment_date**, item_title, bill_description]
[**item_title**, payment_date, bill_description]
[**bill_amount**, payment_date, item_title, bill_description]
[**bill_description**, payment_date, item_title]
```